docs/src/main/paradox/docs/releases/v1.7-to-v1.8-migration.md

```diff
@@ -0,0 +1,447 @@
+ migration | c64c3630-a3a8-11ed-9b31-b3b1660f61b0 |    267972 |         0 |      0 | 2023-02-03 10:54:39.507+01
+```
+
+The table contains the following information:
+
+* `processed`: total number of processed events that have been processed
+* `discarded`: total number of events that have been discarded
+* `failed`:    number of failures risen during the migration
+* `instant`:   timestamp of the last event processed by migration when migration last saved progress
+
+Once the migration terminates, the sum of the processed, discarded, and failed columns should be equal to the number of
+events in your Delta 1.7 deployment.
+
+When looking at the logs, these lines should be also repeated (the timestamp should match the current date/hour):
+
+```
+2023-02-08 12:56:26 INFO  c.e.b.n.m.replay.ReplayEvents - We got 0 rows
+2023-02-08 12:56:26 INFO  c.e.b.n.m.replay.ReplayEvents - Next offset is 1ced2000-a7a8-11ed-8080-808080808080 (2023-02-08 12:00:00:000)
+2023-02-08 12:56:26 INFO  c.e.b.n.m.replay.ReplayEvents - No results for current bucket, waiting for 3000 milliseconds
+```
+
+Keep looking regularly in the logs to look if the migration is still ongoing without problems. If the migration crashes,
+a restart should allow it to start back from where it stopped.
+
+### Post data migration checks
+
+### Internal PostgreSQL consistency check
+
+Once the migration has reached the current date/time, check that the sum of global and scoped events is equal to the
+number of processed events. Running the following query, the two columns should be equal:
+
+```sql
+SELECT
+ (SELECT COUNT(*) FROM scoped_events) +
+ (SELECT COUNT(*) FROM global_events) sum_events,
+ (SELECT processed FROM migration_offset);
+```
+
+### High revision entities check
+
+Find entities that have a high number of revisions:
+
+```sql
+SELECT id, max_rev FROM
+  (SELECT 
+    id, MAX(rev) max_rev 
+    FROM scoped_events 
+    GROUP BY id
+  ) AS idToRev 
+ ORDER BY max_rev DESC LIMIT 5;
+```
+
+and check that their revision counts match the number of revisions in Cassandra:
+
+```sql
+SELECT COUNT(*) FROM delta_1_5.messages WHERE persistence_id = '{persistence_id}' and partition_nr = 0;
+```
+
+where the `persistence_id` is of the form `{entity_type}-{org_name}%2F{project_name}_{id}`. Make sure `{id}` has URL
+encoded characters.
+
+### Automated migration checks
+
+If you are running Delta 1.8 in parallel to Delta 1.7, it is possible to perform automated checks by making POST
+requests to the following Delta 1.8 endpoints once the reindexing is completed:
+
+* `/migration/project-stats`
+    * Count the number of resources in projects across the two instances
+    * Results are written in `public.migration_project_count`
+* `/migration/elasticsearch`
+    * Count the indexed documents in Elasticsearch views across the two instances
+    * Results are written in `public.migration_elasticsearch_count`
+* `/migration/blazegraph`
```

Comment:
We should indicate that we fixed some bugs and introduced new fields that make the count slightly different